server/services/reportedAdjudicationsService.ts

```diff
@@ -25,6 +29,10 @@ export default class ReportedAdjudicationsService {
     private readonly locationService: LocationService
   ) {}
 
+  async getReportedAdjudicationDetails(adjudicationNumber: number, user: User): Promise<ReportedAdjudicationResult> {
+    return new ManageAdjudicationsClient(user.token).getReportedAdjudication(adjudicationNumber)
```

Comment:
What's the difference between this and the below function?

Reply:
so the getConfirmationDetails pulls a bunch of stuff together - like neurodiversity etc. which we need for the non print page. And the getReportedAdjudication just brings back the adjudication - which we need for the actual printed page. I think we might be able to get away with not calling the getConfirmationDetails for the rendering of the pdf - but that will come in a later PR I think